apps/total-typescript/src/hooks/use-mux-player.tsx

```diff
@@ -92,6 +102,18 @@ export const VideoProvider: React.FC<
     }
   }, [subscriber, muxPlayerRef, playbackRate, autoPlay, video])
 
+  const {data: userSession} = useSession()
```

Comment:
load the current session from `next-auth`

this has a status that we need to consider in the overall "loading" status of the context

in the `Video`, we load subscriber again so it might make sense to have all of that logic contained here